src/encoded/tests/test_permissions.py

```diff
@@ -373,21 +369,20 @@ def test_submitter_patch_restricted_assay_fcc_experiment(submitter_testapp, wran
     experiment = {'lab': lab['@id'],
                   'award': award['@id'],
                   'assay_term_name': 'ChIP-seq',
-                  'biosample_ontology': cell_free['uuid'],
-                  'experiment_classification': ['functional genomics assay']}
+                  'biosample_ontology': cell_free['uuid']}
     res = submitter_testapp.post_json('/experiment', experiment, status=201)
     term_name = {'assay_term_name': 'STARR-seq'}
     submitter_testapp.patch_json(res.json['@graph'][0]['@id'], term_name, status=422)
     wrangler_testapp.patch_json(res.json['@graph'][0]['@id'], term_name, status=200)
 
 
-def test_submitter_patch_restricted_classification_fcc_experiment(submitter_testapp, wrangler_testapp, lab, award, cell_free):
-    experiment = {'lab': lab['@id'],
-                  'award': award['@id'],
-                  'assay_term_name': 'ChIP-seq',
-                  'biosample_ontology': cell_free['uuid'],
-                  'experiment_classification': ['functional genomics assay']}
-    res = submitter_testapp.post_json('/experiment', experiment, status=201)
-    classification = {'experiment_classification': ['functional characterization assay']}
-    submitter_testapp.patch_json(res.json['@graph'][0]['@id'], classification, status=422)
-    wrangler_testapp.patch_json(res.json['@graph'][0]['@id'], classification, status=200)
\ No newline at end of file
+#def test_submitter_patch_restricted_classification_fcc_experiment(submitter_testapp, wrangler_testapp, lab, award, cell_free):
```

Comment:
I think you need to delete this test rather than comment it out

Reply:
deleted test_submitter_patch_restricted_classification_fcc_experiment as opposed to commenting out.